homeassistant/components/smartthings/config_flow.py

```diff
@@ -60,39 +49,53 @@ def __init__(self):
         self.api = None
         self.oauth_client_secret = None
         self.oauth_client_id = None
+        self.installed_app_id = None
+        self.refresh_token = None
+        self.location_id = None
 
     async def async_step_import(self, user_input=None):
         """Occurs when a previously entry setup fails and is re-initiated."""
         return await self.async_step_user(user_input)
 
     async def async_step_user(self, user_input=None):
-        """Get access token and validate it."""
+        """Validate and confirm webhook setup."""
+        await setup_smartapp_endpoint(self.hass)
+        webhook_url = f"<blockquote>{get_webhook_url(self.hass)}</blockquote>"
```

Comment:
@balloob don't know if this was OK - but figured we didn't want HTML in the language files themselves?

Reply:
What is it you're trying to do. Why is the blockquote necessary?

Reply:
Looking to have it presented like this:
![Confirmation](https://user-images.githubusercontent.com/6730289/79177946-903c7080-7dc9-11ea-9cc8-956163acc679.PNG)

Reply:
Use `>`

![image](https://user-images.githubusercontent.com/1444314/79275528-38e6e080-7e5b-11ea-8645-1086ce9d79a6.png)